packages/web/src/components/page/FlushPageContainer.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, Flex, FlexProps } from '@audius/harmony'
+
+import { MAX_PAGE_WIDTH_PX, PAGE_GUTTER_PX } from 'common/utils/layout'
+
+export const FlushPageContainer = (props: FlexProps) => {
+  const { children, ...flexProps } = props
+  return (
+    <Flex w='100%' justifyContent='center' {...flexProps}>
+      <Box flex={`0 0 ${PAGE_GUTTER_PX}px`} />
```

Comment:
could this just be padding on the flex container instead of an element?

Reply:
Possibly? I think I noticed some problematic overflow if padding was used as opposed to fixed-width elements that aren't allowed to shrink. I can try it out though, would definitely prefer padding.